cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java

```diff
@@ -62,17 +65,41 @@ public Integer call() throws Exception {
         }
     }
 
-    @Command(name = "remove-service-instance", description = "Removes a service instance.")
+    @Command(name = "remove-service", description = "Removes a service instance.")
     static class RemoveServiceInstanceCommand implements Callable<Integer> {
 
         @Mixin
         LoginCommandOptions loginOptions;
 
         @Mixin
         YamlCommandOptions yamlCommandOptions;
+      
+        @Option(names = { "-f", "--force" }, required = false, description = "Force deletion without confirmation.")
+        Boolean force;
 
         @Override
         public Integer call() throws Exception {
+
+            if (force != null) {
+                doRemoveServiceInstance(yamlCommandOptions);
+
+            } else {
+                
+                System.out.println("Really delete the service y/n?");
```

Comment:
Here, a check is missing whether a terminal is connected on the other end. You can easily hang up a CI job for example with such a prompt, where no terminal is connected to `stdin`, so nobody can answer the question.

This tool should check whether there is no console when `--force` is _not_ used, and output an error message and return a non-zero exit code in that case.